client/fingerprint/fingerprint.go

```diff
@@ -115,6 +116,13 @@ type Fingerprint interface {
 	Periodic() (bool, time.Duration)
 }
 
+// ReloadableFingerprint can be implemented if the fingerprinter needs to be run during client reload.
+// If implemented, the client will call Reload during client reload then immediately Fingerprint
+type ReloadableFingerprint interface {
```

Comment:
Sounds like a nice idea, but this doesn't appear to be implemented by anything?

Reply:
Oops yeah I meant to throw that method on CNI :facepalm: